plugins/coverbrowser.koplugin/bookinfomanager.lua

```diff
@@ -321,6 +321,35 @@ function BookInfoManager:toggleSetting(key)
     return value
 end
 
+function BookInfoManager:getConfigSet()
+    self:openDbConnection()
+    local res = self.db_conn:exec("SELECT key, value FROM config;")
+    local config_set = {}
+    if res then
```

Comment:
(Nitpick: I'll swap lines 326 and 327 for balance - `config_set` created and returned, `res` used in-between. Same below.)